src/components/Homepage/Homepage.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react";
+import { Homepage } from "./Homepage";
+
+describe("Homepage", () => {
+  it("renders dummy text", () => {
+    render(<Homepage />);
+    expect(screen.getByText("Homepage")).toBeInTheDocument();
+  });
+});
```

Comment:
Note the `.test.` in the filename - this is a file that will test our new homepage component.